src/app/shared/utils/lower-time-resolution.ts

```diff
@@ -0,0 +1,12 @@
+import { getTimeWindowEnumValue } from '../enums/time-window.enum'
+
+export function lowerTimeResolution(a: String, b: String) {
+  // NOTE: Return 1 for lower or equal resolution, 0 for higher.
+  // Example: TEN_SEC is lower than ONE_WEEK
+
+  if (getTimeWindowEnumValue(a) <= getTimeWindowEnumValue(b)) {
```

Comment:
this can be a part of `TimeWindow` object. like 
```
object TimeWindow() {
     boolean isLowerTimeResolution(resolutionToCompare) {
           return this <= resolutionToCompare;
     }
}